### PR TITLE
Add Zendesk OAuth credentials doc

### PR DESCRIPTION
**Pull request by harshil1712**

### CODE REVIEW

File: docs/nodes/credentials/Zendesk/README.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ You can find information about the operations supported by the Zendesk node on t
 
 Create a [Zendesk](https://zendesk.com/) account.
 
+## Using OAuth
+
+1. Access your Zendesk dashboard.
+2. Click on the gear icon on the left.
+3. Click on API under the ***Channels*** section.
```
Comment (smamudhan) [inline docs/nodes/credentials/Zendesk/README.md:17]: Please consider adding: "...section, in the sidebar"

File: docs/nodes/credentials/Zendesk/README.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ You can find information about the operations supported by the Zendesk node on t
...
+4. Click on the 'OAuth Clients' tab.
+5. Enter the client name in the ***Client Name*** field.
+6. Enter the description in the ***Description*** field.
+7. Copy the 'OAuth Callback URL' provided in the 'Zendesk OAuth2 API' credentials in n8n and paste it in the ***Redirect URLs*** field in the Zendesk dashboard.
```
Comment (smamudhan) [inline docs/nodes/credentials/Zendesk/README.md:21]: Wouldn't you be pasting this in the Zendesk API credentials page?

File: docs/nodes/credentials/Zendesk/README.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ You can find information about the operations supported by the Zendesk node on t
...
+8. Click on the ***Save*** button.
+9. Use the ***Client Secret*** in the 'Zendesk OAuth2 API in n8n.
```
Comment (smamudhan) [inline docs/nodes/credentials/Zendesk/README.md:23]: Please consider changing to "Use the displayed..."

File: docs/nodes/credentials/Zendesk/README.md [hunk 4]
```diff
@@ -10,6 +10,23 @@ You can find information about the operations supported by the Zendesk node on t
...
+10. Enter the client name in the ***Client ID*** field in n8n.
```
Comment (smamudhan) [inline docs/nodes/credentials/Zendesk/README.md:24]: Please consider adding something like "Enter the client name you used in the Zendesk OAuth clients page"

**Review by smamudhan — CHANGES_REQUESTED** [review]

File: docs/nodes/credentials/Zendesk/README.md (hunk 2 above)
Comment (harshil1712) [reply docs/nodes/credentials/Zendesk/README.md:21]: Nice catch.

**Pull request merged**